### ISSUE TITLE
Nothing happens

### ISSUE THREAD

**Issue by MeAngel** [ShepherdJerred/stTowns#4]
I have  the requirements, Vault, Java 8, MySql (updated the config), and of course MC 1.1 - something.

I see the plugin in the list, there are no errors in console.

I type a command, I see it in chat due to com spy, but nothing happens otherwise:

https://goo.gl/ia2WMk 

Please let me know what else you need

**Comment by ShepherdJerred** [comment]
Can you try removing all other plugins and see if it works then?

**Comment by MeAngel** [comment]
Are there any in particular that you think might conflict, we have 54 something. 

I guess I'll start with the land management plugins and see what happens

**Comment by ShepherdJerred** [comment]
It'd be easier if you just removed them all and see if it worked. After that add them back half at a time and work it down from there.

I'm not sure what plugin would cause a conflict. Does /plot, /nation, or /stto work?

**Comment by MeAngel** [comment]
No luck...   https://goo.gl/XSq4zd

I know on the other server for sure that we're running Java 8 but this is a different host and I"m not sure how to tell.

**Comment by ShepherdJerred** [comment]
Are you not able to remove the .jar files from your /plugins/ directory?

**Comment by MeAngel** [comment]
Yes, confirmed that it's Java8, and yes I can remove the plugins from the directory. When I took a look at the log I realized that I had not set up the MySql settings in the config on this server - I had done it for the other server but not this one. So I added that, uploaded the new config and tried "stto reload" from console but nothing happened at all - so I just restarted the server.

The warnings in the log are repeating perpetually I think so I ended up stopping the server. I think this section is the same set of warnings that are being repeated:

http://pastebin.com/bYPy3TpG

Thanks for your patience

**Comment by ShepherdJerred** [comment]
Same issue as https://github.com/ShepherdJerred/stTowns/issues/3

**Comment by MeAngel** [comment]
Hi I set debug to true and restarted the server:

http://pastebin.com/6yiuSC9P

I dont' use this server a lot so I only accessed (added) MySql this morning with this host. I'm not sure how to tell what version, but I've asked them and will let you know.

**Comment by MeAngel** [comment]
Hi I set debug to true and restarted the server:

http://pastebin.com/6yiuSC9P

I dont' use this server a lot so I only accessed (added) MySql this morning with this host. I'm not sure how to tell what version, but I've asked them and will let you know.

**Comment by ShepherdJerred** [comment]
I'm pretty certain the issue is with MySQL, although it'll spam your error console (and may reduce performance) you shouldn't have an issue with using the plugin if that's the only error.

In a future release I'll remove the errors table unless debug is set to true.

**Comment by MeAngel** [comment]
It'd be nice to not have those errors - it's one of those things that gets under my skin, but the good news is that the plugin does work.

I think when I first tried in my small server that the reason nothing was happening when I typed the commands could have been due to not setting up the database?

And then once I set it up all those warnings started appearing - but now that I have added all the plugins back, I'm actually able to use the plugin so once you release that version that will remove the errors table, I'll move forward with installing this on the other network.

Thanks for your help!
